doc-content/enterprise-only/getting-started-kogito/proc-kogito-custom-spring-boot-project-creation.adoc

```diff
@@ -0,0 +1,96 @@
+[id="proc-kogito-custom-spring-boot-project-creation_{context}"]
+= Creating a custom Spring Boot project for {KOGITO} microservices
+
+You can create custom Maven projects using the Spring Boot archetype for your {KOGITO} microservices. The Spring Boot archetype enables you to add Spring Boot starters or add-ons in the project creation. In {KOGITO}, the following Spring Boot starters are supported:
+
+Processes Spring Boot starter::
+Starter for providing BPMN support to your Spring Boot project. The following is an example of adding a processes Spring boot starter to your project:
+
+[source,xml]
+----
+<dependencies>
+  <dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-processes-spring-boot-starter</artifactId>
+  </dependency>
+</dependencies>
+----
+
+Decisions Spring Boot starter::
+Starter for providing DMN support to your Spring Boot project. The following is an example of adding a decisions Spring boot starter to your project:
+
+[source,xml]
+----
+<dependencies>
+  <dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-decisions-spring-boot-starter</artifactId>
```

Comment:
You are providing specific examples. I think you need to use variables and made the examples generic.

Reply:
I've added more content to the introduction to explain the examples.